src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,48 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the response json with the
+ * aggregated buildbot data.
+ */
+public class BuildInfo {
+  private final String commitHash;
+  private final String timeStamp;
```

Comment:
nit: [timestamp is a single word](https://en.wikipedia.org/wiki/Timestamp). Prefer `timestamp` over `timeStamp` (also in method signatures).